packages/cli/lang/en.lyaml

```diff
@@ -496,6 +496,36 @@ en:
                 describe: "The starting template"
               templateSource:
                 describe: "Path to custom GitHub repository from which to create project template"
+          migrateApp:
+            describe: "Migrate a public app to the projects framework"
+            examples:
+              default: "Migrate a public app to the projects framework"
+            options:
+              appId:
+                describe: "The ID for the public app being migrated to the projects framework"
+              location:
+                describe: "Directory where project should be created"
+              name:
+                describe: "Project name (cannot be changed)"
+            migrationStatus:
+              inProgress: "Converting app configuration to {{#bold}}public-app.json{{/bold}} component definition ..."
+              success: "{{#bold}}Your app was converted and build #1 is deployed{{/bold}}"
+              done: "Converting app configuration to public-app.json component definition ... DONE"
+              failure: "Converting app configuration to public-app.json component definition ... FAILED"
+            warning:
+              title: "{{#bold}}Migrate an app to the projects framework?{{/bold}}"
+              projectConversion: "{{#bold}}The selected app will be permanently converted to a project component.{{/bold}}"
+              appConfig: "All supported app configuration will be moved to the {{#bold}}public-app.json{{/bold}} component definition file. Future updates to those features must be made through the project build and deploy pipeline, not the developer account UI."
+              buildAndDeploy: "This will create a new project with a single app component and immediately build and deploy it to your developer account (build #1)."
+              existingApps: "{{#bold}}This will not affect existing app users or installs.{{/bold}}"
+              copyApp: "We strongly recommend making a copy of your app to test this process in a development app before replacing production."
+            createAppPrompt: "Proceed with migrating this app to a project component?"
+            projectDetailsLink: "View project details in your developer account"
+            errors:
+              invalidAccountType: "Public apps must be migrated within an app developer account. Your current account {{#bold}}{{ accountName }}{{/bold}} is a {{ accountType }}.
+                \n- Run {{#bold}}hs accounts use{{/bold}} to switch your default account to an app developer account.
```

Comment:
we have a `uiCommandReference` for lines like this. It'll help to keep every command reference standard

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1052/commits/62039a6495b1b5eb20d8ddd746508b2daf69a8d4